build.gradle

```diff
@@ -21,23 +27,34 @@ allprojects {
     apply plugin: "idea"
 
     ext {
-        minSdk = 15
-        targetSdk = 27
-        compileSdk = 27
-        buildTools = '28.0.3'
+        minSdkVer = 16
+        targetSdkVer = 30
+        compileSdkVer = 30
+        buildToolsVer = '30.0.2'
 
-        appName = 'gdx-gamesvcs'
-        gdxVersion = '1.9.8'
+        gdxVersion = '1.10.0'
```

Comment:
Is there a problem with 1.9.8 that needs to raise to 1.10.0?

Reply:
No problems with 1.9.8. I just figured that might as well raise the libGdx version to the newest stable version now that it is available. Is it okay to leave as is?

Reply:
For the library, it is better to use the oldest version possible here due to how Gradle works when the library is used within game projects: Gradle will grab the newest version of all declared - so, in order to not change game projects without a need, let's keep 1.9.8.

Reply:
I have reverted the version update back to 1.9.8